Utilities/GenerateTypescriptDTOs/GenerateTypescriptDTOs.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Dapper" Version="2.0.35" />
```

Comment:
I am unsure why Dapper is here, the package shouldn't need it.
I will look into removing it.